tinygrad/tensor.py

```diff
@@ -306,7 +306,10 @@ def min(self, axis=None, keepdim=False): return -((-self).max(axis=axis, keepdim
   def mean(self, axis=None, keepdim=False):
     out = self.sum(axis=axis, keepdim=keepdim)
     return out * (prod(out.shape)/prod(self.shape))
-
+  # TODO: unbiased True 0.01 difference with torch.std
```

Comment:
So it's not ready to merge yet...

Reply:
Can I implement it without the unbiased option, like numpy, just for now @geohot ?

Reply:
The 0.01 error seems to bee an issue with pytorch.

Reply:
That actually makes sense but how?